fluent_comments/views.py

```diff
@@ -36,7 +36,7 @@ def post_comment_ajax(request, using=None):
 
     # Fill out some initial data fields from an authenticated user, if present
     data = request.POST.copy()
-    if request.user.is_authenticated():
+    if request.user.is_authenticated is True:
```

Comment:
This likely fails for Django versions that return `CallableTrue`. It likely needs some compat function instead.

Reply:
Accordingly to Django documentation, this should work: https://docs.djangoproject.com/en/1.11/ref/contrib/auth/#attributes
The tests also pass.

Reply:
Nope: https://code.djangoproject.com/ticket/26988 it clearly states: **Don’t use the is operator for comparisons!**